skyportal/handlers/api/gcn.py

```diff
@@ -598,6 +639,54 @@ async def get(self, dateobs=None):
                     tag_subquery, GcnEvent.dateobs != tag_subquery.c.dateobs
                 )
 
+            if properties_filter is not None:
```

Comment:
I dont see how comma-separated criteria are parsed and used here

Reply:
We do the split and the loop around line 545.